src/ray/util/tests/pipe_logger_test.cc

```diff
@@ -66,56 +57,6 @@ TEST_P(PipeLoggerTest, NoPipeWrite) {
   EXPECT_EQ(actual_content, expected_content);
 }
 
-INSTANTIATE_TEST_SUITE_P(PipeLoggerTest, PipeLoggerTest, testing::Values(1024, 3));
-
-}  // namespace
-
-}  // namespace ray
-
-/////////////////////////////////////////////////
-// Unit test for both unix platform only.
-/////////////////////////////////////////////////
-
-#if defined(__APPLE__) || defined(__linux__)
-
-#include <unistd.h>
-
-namespace ray {
-
-namespace {
-
-TEST_P(PipeLoggerTest, PipeWrite) {
```

Comment:
Actually why removing this test? Shouldn't we just make it run on all platforms?

Reply:
Reason is, the test case is made for (1) pipe write, and check pipe size's effect; (2) log rotation.
Remove because (1) pipe is no longer used in the code; (2) log rotation has been tested elsewhere.